apps/src/templates/Congrats.jsx

```diff
@@ -53,20 +56,26 @@ export default class Congrats extends Component {
     const contentStyle = {
       width: this.responsive.getResponsiveContainerWidth()
     };
+    const store = createStore(combineReducers({responsive: responsiveRedux}));
 
     return (
-      <div style={contentStyle}>
-        <Certificate
-          completedTutorialType={completedTutorialType}
-        />
-        <StudentsBeyondHoc
-          completedTutorialType={completedTutorialType}
-          MCShareLink={MCShareLink}
-          responsive={this.responsive}
-          isRtl={isRtl}
-        />
-        <TeachersBeyondHoc/>
-      </div>
+      <Provider store={store}>
+        <div style={contentStyle}>
+          <Certificate
+            completedTutorialType={completedTutorialType}
+          />
+          <StudentsBeyondHoc
+            completedTutorialType={completedTutorialType}
+            MCShareLink={MCShareLink}
+            responsive={this.responsive}
+            isRtl={isRtl}
+          />
+          <TeachersBeyondHoc
+            responsive={this.responsive}
```

Comment:
Naive, but I thought the idea behind using redux for responsive was that we wouldn't need to pass the responsive parameter down to components any more?

Reply:
That will be the case soon.  I'm working on a separate PR right now that makes it so all of the components on /congrats can access `isRtl` and `responsiveSize` from the redux store.